bats/tests/k8s/traefik.bats

```diff
@@ -76,8 +76,10 @@ assert_traefik_pods_are_up() {
     # Check if the traefik pods come up
     try --max 30 --delay 10 assert_traefik_pods_are_up
     assert_success
-    run curl "http://$(get_host):80"
-    [ "$status" -ne 0 ]
-    run curl -k "https://$(get_host):443"
-    [ "$status" -ne 0 ]
```

Comment:
This didn't make any sense at all before the change — we enable traefik, and then test that it was _not_ available.

Reply:
This change definitely fixes the broken `traefik: enable traefik` test